lib/transport/helpers.ex

```diff
@@ -23,4 +25,23 @@ defmodule Transport.Helpers do
     |> String.split("/")
     |> List.last()
   end
+
+  def format_date(nil), do: ""
+  def format_date(date) do
```

Comment:
Since it this function can raise exception can you add a bang at the end of the name, so it's named `format_date!`

Reply:
Maybe you can rewrite it as : 
```
def format_date(date) do
with {:ok, parsed_date} <- Time.parse("{ISO:Extended}"),
       converted_date <- Timezone.convert("Europe/Paris"),
       {:ok, formated_date} <- Timex.format("{RFC3339}") do
formated_date
else
{:error, error} ->
Logger.error(error)
""
end
end
```